### DISCUSSION TITLE
【水贴】1000丁尼竟然买到了提瓦特坐标

### DISCUSSION THREAD

**Discussion by ran-mitake-ag** [share121/inter-knot#927] [其他]
https://i0.hdslb.com/bfs/new_dyn/f073f447f415f6ffa1c015609646306f1636034895.png@1052w_!web-dynamic.avif
### 还是我，1000买萝卜的那个。
我在这几个小时的探索中不仅出来了，还去提瓦特逛了一圈
**从明天开始我就要把经历都记下来卖5丁尼一章**

> 野外生存课1000丁尼一节`+500丁尼换成提瓦特坐标`
> `2000丁尼`可购买套餐

**Comment by warning-Kirov-reporting** [comment]
你是？...
